components/brave_vpn/browser/connection/brave_vpn_os_connection_api.h

```diff
@@ -133,6 +139,14 @@ class BraveVPNOSConnectionAPI
       mojom::ConnectionState::DISCONNECTED;
   BraveVPNRegionDataManager region_data_manager_;
   base::ObserverList<Observer> observers_;
+  // Used for tracking if the VPN dependencies have been installed.
```

Comment:
Is this really useful? It's going to try to install again at every browser startup so I'm not sure that enabling from two profiles is really worth the extra effort here. I think what would be better here is a one shot event that can be used to track when this is actually ready per my comment about the async install